NEWS.md

```diff
@@ -12,7 +12,10 @@ New language features
     `findall`, `findfirst`, `argmin`/`argmax` and `findmin`/`findmax` to work with these
     objects, returning the index of matching non-missing elements in the parent ([#31008]).
 
-  * `inv(::Missing)` has now been added and returns `missing` ([#31408]).
+  * `inv(::Missing)` has now been added and returns `missing` ([#31451]).
```

Comment:
```suggestion
  * `inv(::Missing)` has now been added and returns `missing` ([#31408]).
```
Rebase artifact? This shouldn't be changing the issue number for a different NEWS entry.

Reply:
Actually, I worked on that issue. And while I was adding `news` entry for the current issue, I realized that I had made a mistake in `inv` issue. That's I made the change.

Reply:
Ah, ok. That should in principle be done in a separate commit but probably not worth it.